lib/matplotlib/sphinxext/plot_directive.py

```diff
@@ -47,6 +47,12 @@
 
 The ``.. plot::`` directive supports the following options:
 
+``:output-base-name:`` : str
```

Comment:
Is this really only the filename base or can it be a relative or absolute path without extension? Either case should be documented. Also, since you start giving users control over the created files, you probably should mention where they go.

Reply:
It's just the file name. The files go where ever Sphinx puts them (it looks like it's in the `_images` directory, but I don't know if that's configurable).

Reply:
Thanks for the clarification. In that case `*-base-name` makes sense.

Two further thoughts:
- would `image-base-name` be better (because more concrete) than `output-base-name`?
- what happens / do we need to check and error if the user tries to give more than a base name, e.g. `:output-base-name: ../escaped_from_image_dir/myimage` or `nested_dir/myimage`?